package.json

```diff
@@ -60,6 +60,10 @@
     "react-addons-perf": "^15.4.2",
     "react-test-renderer": "^15.5.4",
     "redux-mock-store": "^1.2.2",
+    "stylelint": "^8.0.0",
+    "stylelint-order": "^0.6.0",
+    "stylelint-scss": "^2.0.1",
+    "stylelint-value-border-zero": "^1.0.2",
```

Comment:
I've removed `stylelint-order` and `stylelint-value-border-zero` for now. Would rather see this in a separate PR, potentially looking at other plugins at the same time.